std/digest/digest.d

```diff
@@ -991,3 +991,21 @@ unittest
     //necessary, result will have the correct length, but buf will still have it's original
     //length
 }
+
+unittest
```

Comment:
Why? `@system` is the default anyway, `@safe, nothrow, pure, const, @nogc` are not applicable and none of the other tests in `std.digest.digest` have any attributes.